ee/consoleuser/consoleuser_linux.go

```diff
@@ -73,3 +69,26 @@ func CurrentUids(ctx context.Context) ([]string, error) {
 
 	return uids, nil
 }
+
+func listSessions(ctx context.Context) ([]byte, error) {
+	// Try with `--output=json` first, to support the more widely-used older versions of systemd
+	legacyCmd, err := allowedcmd.Loginctl(ctx, "list-sessions", "--no-legend", "--no-pager", "--output=json")
+	if err != nil {
+		return nil, fmt.Errorf("creating loginctl command --no-legend --no-pager --output=json: %w", err)
+	}
+	legacyOut, err := legacyCmd.Output()
+	if err == nil {
+		return legacyOut, nil
+	}
```

Comment:
Hey, OP from https://github.com/kolide/launcher/issues/1758 here. I'm not sure if this will work as intended. I believe the current behavior of `loginctl` is to accept `--output=json` but simply ignore it. In other words, the command does not fail when this (now unrecognized) flag is provided:

```
roobre@Archiroo 󱃾 curry-admin@Curry
14:30:50 ~ $> loginctl list-sessions --output=json && echo Succeeded
SESSION  UID USER   SEAT  LEADER CLASS   TTY IDLE SINCE
      1 1000 roobre seat0 2771   greeter -   no   -
      2 1000 roobre -     3152   manager -   no   -

2 sessions listed.
Succeeded
```

Reply:
Thanks! Still working on getting a test environment with a new enough version of systemd on it to be able to test these changes -- will update this PR and move it out of draft once that's complete 🙂

Reply:
Updated and tested